src/exit/_Exit.rs

```diff
@@ -0,0 +1,8 @@
+use c_types::c_int;
+
+#[no_mangle]
+pub unsafe extern "C" fn _Exit(ec: c_int) -> ! {
+    syscall!(EXIT_GROUP, ec);
+    syscall!(EXIT, ec);
+    loop {}
```

Comment:
Please correct me if I'm wrong, but based on http://git.musl-libc.org/cgit/musl/tree/src/exit/_Exit.c this seems like it should be

```
loop { syscall!(EXIT, ec); }
```

since the C for loop will continue attempting to call the EXIT syscall until the process has aborted.